src/plots/gl3d/scene.js

```diff
@@ -1003,6 +1003,23 @@ function flipPixels(pixels, w, h) {
     }
 }
 
+function correctRGB(pixels, w, h) {
+    for(var i = 0; i < h; ++i) {
+        for(var j = 0; j < w; ++j) {
+            var k = 4 * (w * i + j);
+
+            var a = pixels[k + 3]; // alpha
+            if(a > 0) {
+                var q = 255 / a;
+
+                for(var l = 0; l < 3; ++l) { // RGB
+                    pixels[k + l] = Math.min(q * pixels[k + l], 255);
```

Comment:
Can explain why we need to apply this transform when exporting images?

Reply:
Unlike openGL, webGL `premultipliedAlpha` in enabled by default.
To get correct RGB values from the webgl context it was needed to devide RGB values by alpha.

Reply:
Ok, what would happen if we instead disable `premultipliedAlpha` on image exports?

Reply:
Good question; but that didn't work: [demo](https://codepen.io/MojtabaSamimi/pen/bGddoLJ?editors=1000).